app/src/main/java/kr/ac/konkuk/gdsc/gdscsuyeon/data/TodoDatabase.kt

```diff
@@ -0,0 +1,30 @@
+package kr.ac.konkuk.gdsc.gdscsuyeon.data
+
+import android.content.Context
+import androidx.room.Database
+import androidx.room.Room
+import androidx.room.RoomDatabase
+import kr.ac.konkuk.gdsc.gdscsuyeon.ui.home.HomeFragment
+
+@Database(entities = [Todo::class], version = 1)
+abstract class TodoDatabase: RoomDatabase() {
+    abstract fun todoDao(): TodoDao
+
+    companion object {
+        private var instance: TodoDatabase?= null
+
+        @Synchronized
+        fun getInstance(context: HomeFragment): TodoDatabase? {
```

Comment:
```suggestion
        fun getInstance(context: HomeFragment): TodoDatabase {
```
이렇게 non-null하게 가져올 수 있을 것 같네요!